### PR TITLE
Improvement: Add delay after tool swap before warning about wrong speed

### PR DESCRIPTION
**Pull request by lunaynx**
## What
Sometimes people don't use Rancher's Boots (or just don't have enough base speed without the tool), and tools themselves often give speed, but it takes time for the tab list to update the speed after swapping.

## Changelog Improvements
+ Added a delay after tool swap before warning about wrong farming speed. - Alexia Luna

### CODE REVIEW

**Comment by lunaynx** [comment]
Don't merge yet, looks like with my latest change it actually doesn't work, because you can't draw stuff outside the main thread.

**Comment by lunaynx** [comment]
Should be good to go now.

File: src/main/java/at/hannibal2/skyhanni/features/garden/GardenOptimalSpeed.kt [hunk 1]
```diff
@@ -85,7 +88,9 @@ class GardenOptimalSpeed {
         if (GardenAPI.hideExtraGuis()) return
 
         val text = "Optimal Speed: §f$optimalSpeed"
-        if (optimalSpeed != currentSpeed) {
+        if (lastToolSwitch.passedSince() < 1.5.seconds) {
```
Comment (Thunderblade73) [inline src/main/java/at/hannibal2/skyhanni/features/garden/GardenOptimalSpeed.kt]:
```suggestion
        val colorCode = if (lastToolSwitch.passedSince() < 1.5.seconds) "7" else if (optimalSpeed != currentSpeed) "c" else "a"
```

File: src/main/java/at/hannibal2/skyhanni/features/garden/GardenOptimalSpeed.kt [hunk 2]
```diff
@@ -85,7 +88,9 @@ class GardenOptimalSpeed {
...
+            if (config.showOnHUD) config.pos.renderString("§7$text", posLabel = "Garden Optimal Speed")
```
Comment (Thunderblade73) [inline src/main/java/at/hannibal2/skyhanni/features/garden/GardenOptimalSpeed.kt]:
```suggestion
            config.pos.renderString("§$colorCode$text", posLabel = "Garden Optimal Speed")
```

**Comment by hannibal002** [comment]
How can I reproduce the bug this pr is trying to fix?

**Comment by lunaynx** [comment]
For example, farm Melon/Pumpkin with Fermento Boots where the tool gives you the little extra push to 400. Or Cactus with Cactus Knife and Rancher's Boots set to 464, where the tool uncaps your speed from 400 to 464.

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**